addons/ui/fnc_getInventoryItemData.sqf

```diff
@@ -0,0 +1,58 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Internal Function: cba_ui_fnc_getInventoryItemData
+
+Description:
+    Returns item data from inventory display control
+
+Parameters:
+    _control - RscDisplayInventory control <CONTROL>
+    _index   - Item index <NUMBER>
+
+Returns:
+    _classname, _container, _containerType
+
+Examples:
+    (begin example)
+        [_groundContainer, 5] call cba_ui_fnc_getInventoryItemData;
+    (end)
+
+Author:
+    commy2, SynixeBrett
+---------------------------------------------------------------------------- */
+
+params ["_control", "_index"];
+
+private _unit = call CBA_fnc_currentUnit;
+
+private _container = objNull;
+private _containerType = _control getVariable QGVAR(containerType);
+switch _containerType do {
+    case "GROUND": {
+        _container = GVAR(CurrentGroundItemHolder);
+    };
+    case "CARGO": {
+        _container = GVAR(CurrentContainer);
+    };
+    case "UNIFORM_CONTAINER": {
+        _container = uniformContainer _unit;
+    };
+    case "VEST_CONTAINER": {
+        _container = vestContainer _unit;
+    };
+    case "BACKPACK_CONTAINER": {
+        _container = backpackContainer _unit;
+    };
+};
+
+// Reports classname, but only for magazines.
```

Comment:
When I redid the inventory filtering in https://github.com/acemod/ACE3/pull/9706, I found all item classnames are contained in `lbData` except for backpacks:
https://github.com/acemod/ACE3/blob/b714c8bce2628ff8d4795da8a7f2fc05bb82f474/addons/inventory/functions/fnc_forceItemListUpdate.sqf#L33

Workaround for backpacks is to get them using the display name and picture.

Reply:
> Workaround for backpacks is to get them using the display name and picture.

That would likely cause ambiguities for backpacks with linked items, as I doubt they would change the picture and display name for each variant.